install/kubernetes/helm/istio/templates/crds.yaml

```diff
@@ -0,0 +1,212 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why are these top-level? How do I install mixer without these CRDs/pilot?

Reply:
You can't. And there are no 'mixer' CRDs - those are all Istio CRDs.